geoext.php

```diff
@@ -45,6 +45,7 @@ function GetGeolocationServices()
 {
 	return array(
 		'Yandex',
+		'YandexLocator',
```

Comment:
Disable by default as API key is not specified.
```suggestion
		//'YandexLocator',
```